.idea/misc.xml

```diff
@@ -4,7 +4,7 @@
   <component name="NodePackageJsonFileManager">
     <packageJsonPaths />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="openjdk-1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
Can we be sure that the versions of IDEA the plugin supports will always use a Java 11 runtime? I've been hesitant to upgrade the plugin to use JDK 11, because I cannot find any documentation from JetBrains that definitively tells me that we can expect IDEA to always use a Java 11 runtime.

Reply:
As far as I know IDEAs future is a Java 11 runtime customized by Jetbrains. They call it JBR11. So I think, since Java 11 is the latest LTS we could think about moving towards this version.

Reply:
I did some more digging on this. It looks like JBR11 is not yet guaranteed.

> Starting with 2019.1 version some products are already bundled with JetBrains Runtime version 11 or have the choice between 8 and 11 when downloading. Other products can be switched to JBR11 as described below.

Let's stick with 1.8 until that changes

Reply:
Sorry, I meant to include the link where I found that https://intellij-support.jetbrains.com/hc/en-us/articles/206544879-Selecting-the-JDK-version-the-IDE-will-run-under

Reply:
Okay, in this case we should actually stick with Java 8 👍

Reply:
JetBrains got back to me about this one and clarified with this article https://blog.jetbrains.com/idea/2020/01/intellij-idea-2020-1-eap/